doc/api/buffer.md

```diff
@@ -402,7 +402,7 @@ changes:
 
 > Stability: 0 - Deprecated: Use [`Buffer.from(buffer)`] instead.
 
-* `buffer` {Buffer} An existing `Buffer` to copy data from.
+* `buffer` {Buffer|Uint8Array} An existing `Buffer` or [`Uint8Array`] to copy data from.
```

Comment:
I would make this `{Buffer|TypedArray}` and `An existing `Buffer` or `TypedArray` to copy data from.

Reply:
Still not a fan of that… as you noted in the other PR, this has semantics which match `Buffer.from(array)`, which we document as a *different* overload, and similarly has a completely different performance profile from `Uint8Array`.

I’d be okay with it if we were to merge those two headings together, but that seems out of scope for this PR.

Reply:
> A TypeError will be thrown if array is not an Array.

Well that’s just a plain lie in the docs. :smile: Even `Buffer.from({length: 4})` works perfectly – really all that matters is that `.length` is not `undefined`. If you prefer it, that can also be taken care of at a later point/by somebody else.

Reply:
Probably better to do it later with fresh eyes.  The other Buffer functions should also be revisited.